torch/_dynamo/symbolic_convert.py

```diff
@@ -972,8 +972,10 @@ def load_builtin(self, inst):
         val = self.f_builtins[inst.argval]
 
         if callable(val):
-            assert is_builtin_callable(val)
-            self.push(VariableBuilder(self, GlobalSource(inst.argval))(val))
+            if is_builtin_callable(val):
+                self.push(VariableBuilder(self, GlobalSource(inst.argval))(val))
+            else:
+                self.push(UserFunctionVariable(val, source=GlobalSource(inst.argval)))
```

Comment:
Can you try calling `VariableBullder(val, source=GlobalSource(inst.argval))`? Lets try to ensure that there is a central place for VT building.